service/prometheus/scrapeconfig_test.go

```diff
@@ -119,6 +119,260 @@ func Test_Prometheus_getTarget(t *testing.T) {
 	}
 }
 
+var (
+	TestConfigOne = config.ScrapeConfig{
```

Comment:
We now define the test scrape configs in one place, and refer to them as needed.